config/config.go

```diff
@@ -98,9 +99,12 @@ func (cfg *Config) Load(file string) error {
 			return err
 		}
 		var snippetdirs []string
-		cfg.General.SnippetFile = expandPath(cfg.General.SnippetFile)
 		for _, dir := range cfg.General.SnippetDirs {
-			snippetdirs = append(snippetdirs, expandPath(dir)) // note the = instead of :=
+			if !strings.HasSuffix(dir, "/") {
```

Comment:
Why is this necessary?

Reply:
If I recall correctly, that's because it would lead to new snippet filename being incorrect. 
https://github.com/knqyf263/pet/blob/adfb69a4f874f4d9027176282cc9482eec8afc05/snippet/snippet.go#L94

for example: 

```
/var/tmp/petNewCommand.toml
```

instead of 

```
/var/tmp/pet/NewCommand.toml
```

Reply:
Hmm.. I would say this is a problem on the side of consuming the `snippetdirs` then. When consuming a directory we should make sure to join with os-specific file separators.

I think this is unnecessary if that part works well.

Reply:
I can leave it out for a different PR if that helps